wasmbinding/stargate_allowlist.go

```diff
@@ -47,5 +50,30 @@ func AcceptedStargateQueries() wasmkeeper.AcceptedStargateQueries {
 
 		// feeburner
 		"/neutron.feeburner.Query/Params": &feeburnertypes.QueryParamsResponse{},
+
+		// dex
+		"/neutron.dex.Query/Params":                    &dextypes.QueryParamsResponse{},
+		"/neutron.dex.Query/LimitOrderTrancheUser":     &dextypes.QueryGetLimitOrderTrancheUserResponse{},
+		"/neutron.dex.Query/LimitOrderTranche":         &dextypes.QueryGetLimitOrderTrancheResponse{},
+		"/neutron.dex.Query/UserDepositsAll":           &dextypes.QueryAllUserDepositsResponse{},
+		"/neutron.dex.Query/UserLimitOrdersAll":        &dextypes.QueryAllUserLimitOrdersResponse{},
+		"/neutron.dex.Query/InactiveLimitOrderTranche": &dextypes.QueryGetInactiveLimitOrderTrancheResponse{},
+		"/neutron.dex.Query/PoolReserves":              &dextypes.QueryGetPoolReservesResponse{},
+		"/neutron.dex.Query/EstimateMultiHopSwap":      &dextypes.QueryEstimateMultiHopSwapResponse{},
+		"/neutron.dex.Query/EstimatePlaceLimitOrder":   &dextypes.QueryEstimatePlaceLimitOrderResponse{},
+
+		// incentives
+		"/neutron.incentives.Query/GetModuleStatus":         &incentivestypes.GetModuleStatusResponse{},
+		"/neutron.incentives.Query/GetGaugeByID":            &incentivestypes.GetGaugeByIDResponse{},
+		"/neutron.incentives.Query/GetGauges":               &incentivestypes.GetGaugesResponse{},
+		"/neutron.incentives.Query/GetStakeByID":            &incentivestypes.GetStakeByIDResponse{},
+		"/neutron.incentives.Query/GetStakes":               &incentivestypes.GetStakesResponse{},
+		"/neutron.incentives.Query/GetFutureRewardEstimate": &incentivestypes.GetFutureRewardEstimateResponse{},
+		"/neutron.incentives.Query/GetAccountHistory":       &incentivestypes.GetAccountHistoryResponse{},
+		"/neutron.incentives.Query/GetGaugeQualifyingValue": &incentivestypes.GetGaugeQualifyingValueResponse{},
```

Comment:
you probably've forgotten to add module's params query here?

Reply:
GetModuleStatus includes the params, but I can change it if you prefer @sotnikov-s